GVFS/GVFS.Service/Handlers/NotificationHandler.Mac.cs

```diff
@@ -0,0 +1,27 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+
+namespace GVFS.Service.Handlers
+{
+    public class NotificationHandler

```

Comment:
Does macOS need its own NotificationHandler class? or will there be a shared NotificationHandler base class for Windows and macOS?

Reply:
Yes. Upcoming PR uses it to log notification messages. Am planning to use Platform native notifications (when it becomes available) from here.